site/content/en/docs/Tutorials/nginx_tcp_udp_ingress.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Ingress nginx for TCP and UDP services"
+linkTitle: "Ingress nginx for TCP and UDP services"
+weight: 1
+date: 2019-08-15
+description: >
+  How to set up a minikube ingress for TCP and UDP services
+---
+
+## Overview
+
+
+The Minikube [ingress addon](https://github.com/kubernetes/minikube/tree/master/deploy/addons/ingress) enables developers 
+to route traffic from their Host OS (Laptop, Desktop, etc) to a kubernetes service running inside their minikube cluster.
```

Comment:
* lowercase 'Host'
* I'm not sure that "OS" helps - if you agree, remove it.
* uppercase "kubernetes"